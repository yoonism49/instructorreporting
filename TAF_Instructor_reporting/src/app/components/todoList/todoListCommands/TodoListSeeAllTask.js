// @flow weak

import React from 'react';

const TodoListSeeAllTask = () => (
  <a
    className="btn btn-default btn-sm pull-right"
    href="#">
    See All Tasks
  </a>
);

export default TodoListSeeAllTask;
