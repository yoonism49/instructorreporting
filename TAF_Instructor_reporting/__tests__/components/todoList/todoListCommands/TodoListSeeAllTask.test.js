
'use strict';

import React                from 'react';
import TodoListSeeAllTask   from '../../../../src/app/components/todoList/todoListCommands/TodoListSeeAllTask';
import renderer             from 'react-test-renderer'; // needed both for snpashot testing but also to prevent errors from enzyme

describe('TodoListSeeAllTask component', () => {
  it('renders as expected', () => {
    const component = renderer.create(
      <div>
        <TodoListSeeAllTask />
      </div>
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
