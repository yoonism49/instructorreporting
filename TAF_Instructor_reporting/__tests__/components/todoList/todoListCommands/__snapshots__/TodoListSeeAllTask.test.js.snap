// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoListSeeAllTask component renders as expected 1`] = `
<div>
  <a
    className="btn btn-default btn-sm pull-right"
    href="#"
  >
    See All Tasks
  </a>
</div>
`;
